roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
I think the easier to implement and better to maintain path is to reuse `openshift_facts`, and that's what we're doing now.

Any improvements to the defaulting in `openshift_facts` will be absorbed by us automatically, but I'll leave it out of this PR.